_data/people_list.yml

```diff
@@ -48,18 +56,16 @@
   paid_by_dfg: true
   status: active
 
-- name: Albin Ahlbäck
+- name: "Albin Ahlb\xE4ck"
```

Comment:
I thought this is an artifact of using python's yaml `safe_load`, but this can be fixed by adding `allow_unicode=True` in the dump function. I will open a PR which fixes this.

Reply:
This should be fixed by #395